migrations/Qi.sql

```diff
@@ -1,3 +1,143 @@
 -- 1 up
 
--- 1 down
\ No newline at end of file
+CREATE TABLE if not exists asset_types (
+  asset_type_id int NOT NULL primary key,
```

Comment:
Surely we want our primary ID keys to be auto increment.  In mysql, it used the keyword `auto increment`.  In pg, it uses the special data type 'serial'.  Use `serial` instead of `int`.